src/swaps/liquality/LiqualitySwapProvider.js

```diff
@@ -183,7 +183,7 @@ class LiqualitySwapProvider extends SwapProvider {
 
   updateOrder(order) {
     return axios({
-      url: this.config.agent + '/api/swap/order/' + order.id,
+      url: this.config.agent + '/api/swap/order/' + order.orderId,
```

Comment:
Is this fixing of old bug? 
If we create a boost provider in the perfect case we should not change the liquality provider, just use it as it is.
If it is needed then it is fine, but be careful for regressions.

Reply:
It's not a bug. It's the only way to make things work with minimal changes (that came to my mind).
`orderId` and `id` are the same thing. But in case of swapping from erc20 to native we need to keep the same `id` between bot swaps (the one over AMM and over LSP). Otherwise wallet cant pull the correct swap data from storage.